architecture/service_catalog/service_catalog_cli.adoc

```diff
@@ -0,0 +1,446 @@
+=== Deleting service bindings
+
+. To delete all service bindings, associated with a service instance:
++
+[source, bash]
+----
+$ svcat unbind -n <project-name> <1>
+  \ <instance-name> <2>
+----
+<1> Name of the project that contains the service instance.
+<2> Name of the service instance associated with the binding.
+
++
+For example:
++
+[source, bash]
+----
+$ svcat unbind -n szh-project postgresql-instance
+deleted mediawiki-postgresql-binding
+
+$ svcat get bindings
+  NAME   NAMESPACE   INSTANCE   STATUS
++------+-----------+----------+--------+
+
+----
+
++
+[NOTE]
+====
+Running this command deletes all service bindings for the instance. For deleting individual bindings from within an instance run the command `svcat unbind -n <project-name> --name <binding-name>`. For example, `svcat unbind -n szh-project --name mediawiki-postgresql-binding`.
+====
+
+. Verify that the associated secret is deleted.
++
+[source, bash]
+----
+$ oc get secret -n szh-project
+NAME                       TYPE                                  DATA      AGE
+builder-dockercfg-jxk48    kubernetes.io/dockercfg               1         9m
+builder-token-92jrf        kubernetes.io/service-account-token   4         9m
+builder-token-b4sm6        kubernetes.io/service-account-token   4         9m
+default-dockercfg-cggcr    kubernetes.io/dockercfg               1         9m
+default-token-g4sg7        kubernetes.io/service-account-token   4         9m
+default-token-hvdpq        kubernetes.io/service-account-token   4         9m
+deployer-dockercfg-wm8th   kubernetes.io/dockercfg               1         9m
+deployer-token-hnk5w       kubernetes.io/service-account-token   4         9m
+deployer-token-xfr7c       kubernetes.io/service-account-token   4         9m
+----
+
+[[service-catalog-cli-delete-serviceinstance]]
+=== Deleting service instances
+
+Deprovision the service instance:
+
+[source, bash]
+----
+$ svcat deprovision postgresql-instance
+deleted postgresql-instance
+
+$ svcat get instance
+  NAME   NAMESPACE   CLASS   PLAN   STATUS
++------+-----------+-------+------+--------+
+----
+
+[[service-catalog-cli-delete-servicebrokers]]
+=== Deleting service brokers
+. To remove broker services for the service catalog, delete the `ClusterServiceBroker` resource:
++
+[source, bash]
+----
+$ oc delete clusterservicebrokers template-service-broker
+clusterservicebroker "template-service-broker" deleted
+
+$ svcat get brokers
+           NAME                                                        URL                                              STATUS
++-------------------------+-------------------------------------------------------------------------------------------+--------+
+  ansible-service-broker    https://asb.openshift-ansible-service-broker.svc:1338/ansible-service-broker                Ready
+----
+
+. View the `ClusterServiceClass` resources for the broker, to verify that the broker is removed:
++
+[source, bash]
+----
+$ svcat get classes
```

Comment:
The output should be as below, uuid have been removed. 
```
$ svcat get classes
  NAME   DESCRIPTION
+------+-------------+
```

Reply:
Thanks @sunzhaohua2 I have updated this.